.github/workflows/CI.yml

```
@@ -71,13 +71,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
-          version: '1'
+          version:
+            - '1'
+            - '1.6'
```

```suggestion
            - '1'
```
Sorry, I thought this was part of the main testing bit rather than the docs 🤦 . Version `1` is fine for building the docs.

---

Fixed, resolving